src/app.ts

```diff
@@ -208,6 +208,12 @@ async function app() {
     ) ?? [];
   const fhirPackageId = determineCorePackageId(config.config.fhirVersion[0]);
   allDependencies.push(`${fhirPackageId}@${config.config.fhirVersion[0]}`);
+  await utils.loadAutomaticDependencies(
+    config.config.fhirVersion[0],
+    config.config.dependencies ?? [],
+    // @ts-ignore TODO: this can be removed once SUSHI changes the type signature for this function to use FPL's FHIRDefinitions type
```

Comment:
Did we say there should be a PR for SUSHI to do this as part of this overall task (but no release just for this)?

Reply:
We did! Just created [CIMPL-1143](https://standardhealthrecord.atlassian.net/browse/CIMPL-1143) for this.

Reply:
Oh nice, I lost track of that task! Thanks!